share/abbrv.bib

```diff
@@ -27,9 +29,12 @@ @string{infjoc
 @string{jacm = {J. Assoc. Comput. Mach.}}
 @string{jasa = {J. Am. Stat. Assoc.}}
 @string{jcam = {J. Comput. Appl. Math.}}
+@string{jcgs = {J. Comput. Graphical Stat.}}
 @string{jcm = {J. Comput. Math.}}
 @string{jco = {J. Complexity}}
 @string{jcp = {J. Comput. Phys.}}
+@string{jcvxa = {J. Conv. Anal.}}
```

Comment:
```suggestion
@string{jcvxa = {J. Convex Anal.}}
```

Reply:
This one was here: https://images.webofknowledge.com/images/help/WOS/A_abrvjt.html